src/views/VideoView/VideoView.tsx

```diff
@@ -31,6 +31,24 @@ const VideoView: React.FC<RouteComponentProps> = () => {
 
   const videoID = data?.video?.id
 
+  const [playing, setPlaying] = useState<boolean>(true)
+  const handleUserKeyPress = useCallback((event) => {
```

Comment:
We should type `event` here, ideally that would be `event: KeyboardEvent` but `window.addEventListener` won't accept that unfortunately so we will need to use `event: Event`